### ISSUE TITLE
when i run 'main.dart'

### ISSUE THREAD

**Issue by lixinyang8906** [flutter/flutter#26619]
Could not find an option named "preview-dart-2".

Run 'flutter -h' (or 'flutter <command> -h') for available flutter commands and options.

**Comment by lixinyang8906** [comment]
![image](https://user-images.githubusercontent.com/8587899/51225869-a588dc00-1987-11e9-8e26-1f8c06fc58d3.png)

Just click the button to display the error

**Comment by lixinyang8906** [comment]
![image](https://user-images.githubusercontent.com/8587899/51225905-d406b700-1987-11e9-832a-d64e73cf82bf.png)

Just click the run to display the error

**Comment by lixinyang8906** [comment]
![image](https://user-images.githubusercontent.com/8587899/51226659-5b095e80-198b-11e9-9abc-ebbd996141cf.png)

Am I in the right place to execute this command?

**Comment by amatshiye** [comment]
No. You need to be inside of your project to execute the command.

**Comment by lixinyang8906** [comment]
too much information!!!!!!!!!

**Comment by lixinyang8906** [comment]
![image](https://user-images.githubusercontent.com/8587899/51228501-d111c380-1993-11e9-9655-efde3b548dc0.png)

**Comment by lixinyang8906** [comment]
![image](https://user-images.githubusercontent.com/8587899/51228529-f0105580-1993-11e9-9188-919bae69633b.png)

That's just a little bit.

**Comment by amatshiye** [comment]
Are you inside your project folder now?

**Comment by lixinyang8906** [comment]
![image](https://user-images.githubusercontent.com/8587899/51228599-2a79f280-1994-11e9-8728-a34843e96a86.png)
 

yes！！！

**Comment by amatshiye** [comment]
Okay cool. Is there still an error about "preview-dart-2" ?

**Comment by lixinyang8906** [comment]
Yes. This problem will arise no matter what I do.So does redownloading the SDK

**Comment by amatshiye** [comment]
Run flutter run and show me the error.

**Comment by lixinyang8906** [comment]
When I excute the command, It works in the emulator, but clicking the run button in android studio still brings up the error.
![image](https://user-images.githubusercontent.com/8587899/51228870-2ef2db00-1995-11e9-9e8d-5c6473ab5b10.png)

**Comment by amatshiye** [comment]
I don't use android studio for flutter. Can you try Visual Studio Code since it has the debug buttons too and see if the error is there.

**Comment by lixinyang8906** [comment]
I don't have Visual Studio................................................

**Comment by amatshiye** [comment]
Would you be able to download it now?

**Comment by kangwang1988** [comment]
@lixinyang8906 
Paste `flutter doctor -v` here.
Besides, Check if your flutter plugin(Android Studio) is updated to the latest version. In the older version, there is a checkbox for preview-dart-2.
You can also search your flutter project to check if there is some flag `preview-dart-2`. This problem occurs because that some place(Android Studio's flutter plugin, or local.properties,etc) has set the `preview-dart-2` flag. Easy to fix, but you have to check where it's set.

**Comment by lixinyang8906** [comment]
@kangwang1988 
![image](https://user-images.githubusercontent.com/8587899/51236457-27d7c700-19ad-11e9-8d61-ab1cc2acab92.png)

All of these are the latest version.

**Comment by lixinyang8906** [comment]
I don't want to solve it, I can't solve it, I can run it in command.
